src/Command.php

```diff
@@ -322,10 +337,18 @@ public function execute()
                 1   => array('pipe','w'),
                 2   => array('pipe', $this->getIsWindows() ? 'a' : 'w'),
             );
+            if (isset($this->_stdIn)) {
```

Comment:
I usually prefer `!== null` instead of `isset()` as I find it more clear. Could you maybe change that?

Reply:
Glad that you liked it! 
I have changed this as requested.